File: Modul5/Python/PRAK503-2310817210020-MuhammadHasbiyanRusyadi.py
def max(a,b) :
    if (a > b) :
        return a
    else :
        return b
def min(a,b) :
    if (a < b) :
        return a
    else :
        return b
batas = 0
maks = -100000
minim = 100000
angka = int(input())
nilai = list(map(int, input().split()))
while (batas < angka) :
    maks = max(maks,nilai[batas])
    minim = min(minim,nilai[batas])
    batas += 1  
print(f"{maks} {minim}")

File: Modul5/C/PRAK503-2310817210020-MuhammadHasbiyanRusyadi.c
#include <stdio.h>
int maksimal(int x, int y){
    if (x > y) {
    return x;}
    else  {
    return y;}}
int minimal(int x, int y){
    if (x < y) {
    return x;}
    else {
    return y;}}
int main(){
    int batas = 0;
    int max = -100000;
    int min = 100000;
    int angka;
    scanf("%d", &angka);
    while(batas < angka){
      int nilai;
        scanf("%d", &nilai); 
        max = maksimal(max, nilai);
        min = minimal(min, nilai);
        batas++;}
    printf("%d %d",max,min); }

